scripts/reupload_databases.py

```diff
@@ -227,7 +230,7 @@ def getProgDbgAsJson(docLink, saveToFile=False):
     return formatted_dict
 
 def checkProgrammerToDevice(database, devices, progDbgInfo, addGeneral=False):
-    ProgrammerToDeviceColumns = 'programer_uid, device_uid'
+    ProgrammerToDeviceColumns = 'programer_uid, device_uid, device_support_package'
```

Comment:
Check for live database. Do we have 3 columns there too?

Reply:
No, ifdefed for test index